docs/source/quantization.rst

```diff
@@ -1341,6 +1341,7 @@ Please take a look at `Limitations of Symbolic Tracing <https://pytorch.org/docs
 .. py:module:: torch.ao.quantization.pt2e.qat_utils
 .. py:module:: torch.ao.quantization.pt2e.representation.rewrite
 .. py:module:: torch.ao.quantization.pt2e.utils
+.. py:module:: torch.ao.quantization.pt2e.lowering
```

Comment:
```suggestion
.. py:module:: torch.ao.quantization.pt2e.lowering.lower_pt2e_quantized_to_x86
```

maybe try this

Reply:
Thanks. I will try.

Reply:
It does not work. The error is now:
```
module torch.ao.quantization.pt2e.lowering.lower_pt2e_quantized_to_x86 could not be imported: No module named 'torch.ao.quantization.pt2e.lowering.lower_pt2e_quantized_to_x86'; 'torch.ao.quantization.pt2e.lowering' is not a package
```

Reply:
Hi @jerryzh168 Do you have more suggestions? Thanks.

Reply:
oh sorry, then we should revert back to `torch.ao.quantization.pt2e.lowering` I think

Reply:
one thing to try is to add to https://github.com/pytorch/pytorch/blob/31634b8c6ac5ae25e792c0c98cb35209d14367eb/docs/source/quantization-support.rst#torchaoquantizationpt2e-quantization-in-pytorch-20-export-implementation

you can pass CI by adding it to https://github.com/pytorch/pytorch/blob/31634b8c6ac5ae25e792c0c98cb35209d14367eb/docs/source/conf.py#L307 but I think it's not recommended (from @albanD), so we'll need a proper fix here.

Reply:
Thanks. It seems working.